README.md

```diff
@@ -1 +1,284 @@
+    - This script will run a graph query to get recommendations for item
+      `peanut butter`. You can change the item name in the script to get
+      recommendations for other items.
+
+### Add additional data (New vertex and edge)
+
+
+> **Note**: Please make sure that you are running commands in order.
+
+
+1. Run `create/create-vertex.js` to add new vertex in Macrometa GDN.
+    - This script will add a one new vertex in `groceryCustomers` and `groceryItems` collection.
+    - Example data in `groceryCustomers`: `{_key: "C16", name: "John Harris"}`
+    - Example data in `groceryItems`: `{_key: "P21", name: "Avocado"}`
+
+2. Run `create/create-edge.js` to add new edge in Macrometa GDN.
+    - This script will add a one new edge in `groceryOrders` collection.
+    - Example data in `groceryOrders`: `{_key: "4117657859", _from: "groceryCustomers/C16", _to: "groceryItems/P21"}`
+
+
+> **Note**: If you run this scripts multiple times you will get and error. This is because we are using same vertex and edge name. 
+You can change the name of vertex and edge in the script to avoid this error and add new data to graph.
+To run this code without an error you need to delete the vertex and edge from graph first. (If using same name)
+
+
+## ❓ Query explanation
+
+@name is a bind variable that can be set to any item name. In this example we
+have set it to `peanut butter`. First we are filtering by item name and then
+collecting all the users who purchased that item. Then we are collecting all the
+items that were purchased by those users. We are filtering out the item that we
+are looking (`peanut butter`) for and sorting the result by the number of times
+it was purchased.
+
+### Query
+
+```
+// Query: Find the most popular item in the grocery store based on the number of users who purchased it
+LET doc = (FOR d in groceryItems FILTER d.name == @name RETURN d)[0]
+FOR user IN 1..1 INBOUND doc groceryOrders
+    FOR item IN 1..1 OUTBOUND user groceryOrders
+        COLLECT i=item.name WITH COUNT INTO c
+        FILTER i != @name
+        SORT c DESC
+        // LIMIT 2
+        RETURN {"item": i, "count": c}   // item -> number of users who purchased it
+                                         // count -> number of times it was purchased
+```
+
+### Result
+
+If bind variable `@name` is set to `peanut butter` then result will be:
+
+```
+[
+  { item: 'jelly', count: 2 },
+  { item: 'napkins', count: 2 },
+  { item: 'strawberry', count: 2 },
+  { item: 'banana', count: 1 },
+  { item: 'milk', count: 1 },
+  { item: 'celery', count: 1 },
+  { item: 'bread', count: 1 },
+  { item: 'white claw', count: 1 },
+  { item: 'kiwi', count: 1 },
+  { item: 'ketchup', count: 1 },
+  { item: 'apple', count: 1 }
+]
+```
+
+First recommendation according to this query is `jelly` which is purchased by 2
+users. Second recommendation is `napkins` which is purchased by 2 users. Third
+recommendation is `strawberry` which is purchased by 2 users.
+
+## Dataset
```

Comment:
What is going on here???

Reply:
Cnaged the title to make it clear, it is good practice to show developers what kind of test data they are using in the example.